types.go

```diff
@@ -14,3 +14,7 @@ type Result struct {
 	Measurements MeasurementsMap `json:"measurements"`
 	Message      interface{}     `json:"message"`
 }
+
+func floatToIntPercent(f float64) int{
+	return int(f*100+0.5)
+}
```

Comment:
I think it's good practice to save files with a newline at the end

Reply:
fixed